src/uiprotect/api.py

```diff
@@ -153,6 +153,14 @@ def get_user_hash(host: str, username: str) -> str:
     return session.hexdigest()
 
 
+def dict_from_unifi_list(self, list) -> dict[str, Any]:
```

Comment:
Maybe move `dict_from_unifi_list` into `convert.py` near `create_from_unifi_dict` and call it `unifi_list_to_unifi_dict`